features/pages/cscc_page.rb

```diff
@@ -0,0 +1,24 @@
+require 'page-object'
+
+class HomePage
+  include PageObject
+
+  page_url "http://cscc.edu/"
+
+  link(:results, :class => "gs-title")
+
+
+  def input_search
+  	browser.text_field(:id => "input").set "csci 2994"

```

Comment:
Should have defined your text_field on the page object itself, instead of calling it on watir-webdriver.